.circleci/config.yml

```diff
@@ -123,28 +164,6 @@ workflows:
                 - master
 
       # UNIX tests
-      - test-unix:
-          name: Unix Tests for Node=12
-          context: nodejs-install
-          node_version: '12.22'
-          requires:
-            - Lint
-          filters:
-            branches:
-              ignore:
-                - master
-
-      - test-unix:
-          name: Unix Tests for Node=14
-          context: nodejs-install
-          node_version: '14.20'
-          requires:
-            - Lint
-          filters:
-            branches:
-              ignore:
-                - master
-
```

Comment:
Waiting for https://github.com/snyk/snyk-nuget-plugin/pull/153 to be merged. So ignore in this PR.